src/libCrypto/EthCrypto.cpp

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (C) 2022 Zilliqa
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "EthCrypto.h"
+
+#include <openssl/ec.h>  // for EC_GROUP_new_by_curve_name, EC_GROUP_free, EC_KEY_new, EC_KEY_set_group, EC_KEY_generate_key, EC_KEY_free
+#include <openssl/obj_mac.h>  // for NID_secp192k1
+#include <openssl/sha.h>      //for SHA512_DIGEST_LENGTH
+#include <ethash/keccak.hpp>
+#include <functional>
+#include <iostream>
+#include <memory>
+#include <string>
+
+using namespace std;
```

Comment:
Most  style guides I saw forbid `using namespace` even in .cpp files.  You could probably use `using std::cout` etc.  Or just remove it as you are using `std::` qualifiers everywhere explicitly anyways.

Reply:
That's fair. I will remove it - although there are 100 other files in the codebase that already do this!